03-setup-ingress-controller/guides/wildcard_certificates.md

```diff
@@ -0,0 +1,682 @@
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt-dev-wcard
+  namespace: dev
+spec:
+  # ACME issuer configuration:
+  # `email` - the email address to be associated with the ACME account (make sure it's a valid one).
+  # `server` - the URL used to access the ACME server’s directory endpoint.
+  # `privateKeySecretRef` - Kubernetes Secret to store the automatically generated ACME account private key.
+  acme:
+    email: my.email@yahoo.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      name: letsencrypt-dev-wcard-private
+    # List of challenge solvers that will be used to solve ACME challenges for the matching domains.
+    solvers:
+    # Use the DigitalOcean DNS API to manage DNS01 challenge records.
+      - dns01:
+          digitalocean:
+            # Kubernetes secret that contains the DO API token.
+            # Must be in the same namespace as the Issuer CRD.
+            tokenSecretRef:
+              name: digitalocean-dns
+              key: access-token
+```
+
+First, change directory where the `Starter Kit` repository was cloned on your local machine:
+
+```shell
+cd Kubernetes-Starter-Kit-Developers
+```
+
+Then, open and inspect the `Issuer` manifest file provided in the `Starter Kit` repository, using an editor of your choice (preferably with `YAML` lint support). For example, you can use [VS Code](https://code.visualstudio.com). Please pick only one option, depending on the installed `Ingress Controller` (make sure to replace the `<>` placeholders accordingly, using a **valid e-mail address**):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  code 03-setup-ingress-controller/assets/manifests/cert-manager-ambassador-wcard-issuer.yaml
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  code 03-setup-ingress-controller/assets/manifests/cert-manager-nginx-wcard-issuer.yaml
+  ```
+
+Save the file and apply changes for your `Kubernetes` cluster using `kubectl` (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  kubectl apply -f 03-setup-ingress-controller/assets/manifests/cert-manager-ambassador-wcard-issuer.yaml
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl apply -f 03-setup-ingress-controller/assets/manifests/cert-manager-nginx-wcard-issuer.yaml
+  ```
+
+Verify `Issuer` status using `kubectl` (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  kubectl get issuer letsencrypt-ambassador-wcard -n ambassador
+  ```
+
+  The output looks similar to (notice the `READY` column value - should be `True`):
+
+  ```text
+  NAME                           READY   AGE
+  letsencrypt-ambassador-wcard   True    3m32s
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl get issuer letsencrypt-nginx-wcard -n ingress-nginx
```

Comment:
The issuer is created in the backend namespace.
The command should be:
kubectl get issuer letsencrypt-nginx-wcard -n backend

Reply:
Should be fixed in the latest commit. Thanks.